eppo-sdk-test/validators/RuleValidatorTest.cs

```diff
@@ -202,6 +209,23 @@ private static void AddNumericConditionToRule(Rule rule)
         });
     }
 
+    private static void AddSemVerConditionToRule(Rule rule)
+    {
+        rule.conditions.Add(new Condition
+        {
+            value = new EppoValue("1.0.0", EppoValueType.STRING),
```

Comment:
consider changing this to `1.5.0` so we **know** true semver comparison is being made vs string comparison